Composer/babel.l10n.config.js

```diff
@@ -1,13 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// TODO: It would be nice to use the regular expression matching.
+// It would be nice to ignore the lib folders rather than listing them one at a time.
+// However, as of 4/2021 it doesn't seem to work.
+// We have the added complexity that packages/lib contains folders we DO want to process.
```

Comment:
Yeah. Easiest thing to do is probably just remove the lib directory and move the contents up a directory.

Reply:
Agree.  Didn't want to create any major restructure so close to release.

Reply:
I definitely agree here, too. It keeps tripping me up that we have regular non-compiled code in a `lib` folder (and then each package in there has compiled stuff in its own `lib` folder).